pkg/controllers/resources/pods/validatePodSecurity.go

```diff
@@ -0,0 +1,110 @@
+package pods
+
+import (
+	"context"
+	"fmt"
+
+	admissionv1 "k8s.io/api/admission/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	kubeinformers "k8s.io/client-go/informers"
+	"k8s.io/pod-security-admission/admission"
+	admissionapi "k8s.io/pod-security-admission/admission/api"
+	"k8s.io/pod-security-admission/api"
+	"k8s.io/pod-security-admission/metrics"
+	"k8s.io/pod-security-admission/policy"
+)
+
+func validatePodSecurityStandards(ctx context.Context, pod *corev1.Pod, s *podSyncer) (*admissionv1.AdmissionResponse, error) {
```

Comment:
please don't put the whole podSyncer as parameter, instead make the function part of the podSyncer with:
```
func (s *podSyncer) validatePodSecurityStandards(ctx context.Context, pod *corev1.Pod) (*admissionv1.AdmissionResponse, error) {
```

Reply:
Okay